pandas/io/formats/style_render.py

```diff
@@ -1418,7 +1532,7 @@ def _default_formatter(x: Any, precision: int, thousands: bool = False) -> Any:
     """
     if isinstance(x, (float, complex)):
         return f"{x:,.{precision}f}" if thousands else f"{x:.{precision}f}"
-    elif isinstance(x, int):
+    elif isinstance(x, (int, np.int64)):
```

Comment:
Think you can use `from pandas.core.dtypes.common import is_integer`

Reply:
done